index.html

```diff
@@ -195,6 +195,11 @@
     <script src="src/spatialCursor/shader/colorCursorFragmentShader.js" type="module"></script>
     <script src="src/spatialCursor/shader/vertexShader.js" type="module"></script>
 
+    <script src="src/spatialCapture/Shaders.js" type="module"></script>
+    <script src="src/spatialCapture/VisualDiff.js" type="module"></script>
+    <script src="src/spatialCapture/CameraVisPatch.js" type="module"></script>
+    <script src="src/spatialCapture/SpatialPatchCoordinator.js" type="module"></script>
```

Comment:
Since SpatialPatchCooordinator imports CameraVisPatch which imports the other two files only the SpatialPatchCoordinator <script> tag is needed here